### PR TITLE
Update home page

### PR DESCRIPTION
**Pull request by vpchung**
This PR will update the HTML and SCSS of the home page components:

- [x] challenge-search
- [x] statistics-viewer
- [x] topics-viewer
- [x] challenge-host-list
- [x] challenge-registration
- [x] featured-challenge-list
- [x] sponsor-list

Note: I'm not sure how to implement `@media` into Angular Material Theme yet... for now, the `@media` typography will remain in `home.scss`.

### CODE REVIEW

**Comment by vpchung** [comment]
`topics-viewer` component update finished:

**Before**
Screen Shot 2022-06-30 at 1 02 44 PM

**After**
Screen Shot 2022-06-30 at 1 20 28 PM

**Comment by vpchung** [comment]
`challenge-host-list` component update finished:

**Before**
Screen Shot 2022-06-30 at 1 02 49 PM

**After**
Screen Shot 2022-06-30 at 1 28 08 PM

**Comment by vpchung** [comment]
`challenge-registration` component update finished:

**Before**
Screen Shot 2022-06-30 at 1 02 55 PM

**After**
Screen Shot 2022-06-30 at 2 20 04 PM

**Comment by vpchung** [comment]
`challenge-registration` component update finished:

**Before**
Screen Shot 2022-06-30 at 1 02 55 PM

**After**
Screen Shot 2022-06-30 at 2 54 00 PM

**Comment by vpchung** [comment]
`featured-challenge-list` component update finished:

**Before**
Screen Shot 2022-06-30 at 1 03 02 PM

**After**
Screen Shot 2022-06-30 at 3 59 28 PM

**Comment by vpchung** [comment]
`sponsor-list` component update finished:

**Before**
Screen Shot 2022-06-30 at 1 03 08 PM

**After**
Screen Shot 2022-06-30 at 4 25 10 PM

**Comment by vpchung** [comment]
@tschaffter @rrchai I finished updating the home page so that the typography is displayed among the different components.  There are some colors that remain in the SCSS, but I will tackle that in my next PR (where I will restructure where all of the themes/styles go).

File: libs/challenge-registry/home/src/lib/challenge-host-list/challenge-host-list.component.html [hunk 1]
```diff
@@ -1,43 +1,41 @@
...
-          <h4>Sage Bionetworks</h4>
-          <div class="card-footer">
-            <a href="#">@sage</a>
-          </div>
+    <h3 class="section-title">Discover challenge hosts</h3>
+    <div class="row">
+      <!-- TODO: replace hard-coded cards with ngFor -->
+      <div class="col org-card text-center">
+        <img
+          src="https://raw.githubusercontent.com/Sage-Bionetworks/rocc-app/7e8c4f27849796ce52df74118de32b3a6f1159e1/images/logo/sage-bionetworks.png"
```
Comment (tschaffter) [inline libs/challenge-registry/home/src/lib/challenge-host-list/challenge-host-list.component.html:8]: We should add this image to the challenge-registry-assets project.

File: libs/challenge-registry/home/src/lib/challenge-host-list/challenge-host-list.component.html [hunk 2]
```diff
@@ -1,43 +1,41 @@
...
-            src="https://raw.githubusercontent.com/Sage-Bionetworks/rocc-app/7e8c4f27849796ce52df74118de32b3a6f1159e1/images/logo/miccai.png"
-            alt="miccai">
-          <h4>MICCAI</h4>
-          <div class="card-footer">
-            <a href="#">@miccai</a>
-          </div>
+      </div>
+      <div class="col org-card text-center">
+        <img
+          src="https://raw.githubusercontent.com/Sage-Bionetworks/rocc-app/7e8c4f27849796ce52df74118de32b3a6f1159e1/images/logo/miccai.png"
```
Comment (tschaffter) [inline libs/challenge-registry/home/src/lib/challenge-host-list/challenge-host-list.component.html:17]: We should add this image to the challenge-registry-assets project.

File: libs/challenge-registry/home/src/lib/challenge-host-list/challenge-host-list.component.html [hunk 3]
```diff
@@ -1,43 +1,41 @@
...
-            src="https://raw.githubusercontent.com/Sage-Bionetworks/rocc-app/7e8c4f27849796ce52df74118de32b3a6f1159e1/images/logo/cagi.png"
-            alt="cagi">
-          <h4>CAGI</h4>
-          <div class="card-footer">
-            <a href="#">@cagi</a>
-          </div>
+      </div>
+      <div class="col org-card text-center">
+        <img
+          src="https://raw.githubusercontent.com/Sage-Bionetworks/rocc-app/7e8c4f27849796ce52df74118de32b3a6f1159e1/images/logo/cagi.png"
```
Comment (tschaffter) [inline libs/challenge-registry/home/src/lib/challenge-host-list/challenge-host-list.component.html:26]: We should add this image to the challenge-registry-assets project.

File: libs/challenge-registry/home/src/lib/challenge-host-list/challenge-host-list.component.html [hunk 4]
```diff
@@ -1,43 +1,41 @@
...
-        <div class="col org-card text-center">
-          <img src="https://raw.githubusercontent.com/Sage-Bionetworks/rocc-app/main/images/logo/precisionfda.png"
-            alt="precisionfda">
-          <h4>PrecisionFDA</h4>
-          <div class="card-footer">
-            <a href="#">@precisionfda</a>
-          </div>
+      </div>
+      <div class="col org-card text-center">
+        <img src="https://raw.githubusercontent.com/Sage-Bionetworks/rocc-app/main/images/logo/precisionfda.png"
```
Comment (tschaffter) [inline libs/challenge-registry/home/src/lib/challenge-host-list/challenge-host-list.component.html:34]: We should add this image to the challenge-registry-assets project.

**Review by tschaffter — CHANGES_REQUESTED** [review]

**Comment by tschaffter** [comment]
> Note: I'm not sure how to implement @media into Angular Material Theme yet... for now, the @media typography will remain in home.scss.

Sounds good! Let's track this in a ticket that we can revisit in the future.

**Review by rrchai — APPROVED** [review]

**Pull request merged**